guides/flex-layout.md

```diff
@@ -59,29 +59,27 @@ good primer.</aside>
 
 ### Using layout classes
 
-To use layout classes import the `iron-flex-layout-classes` file. You
-must do this in any element that uses any of the `iron-flex-layout` styles.
+The `iron-flex-layout-classes` file provides a set of style modules, so you can include only
+the classes you need. To use the classes, you must import the file, then include the style
+modules you want.
 
-    <link rel="import" href="bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+1.  Import `iron-flex-layout-classes.html`:
 
-Then include the module(s) that you need:
+        <link rel="import" href="bower_components/iron-flex-layout/iron-flex-layout-classes.html">
 
-    <!-- include classes in the main document -->
-    <style is="custom-style" include="iron-flex iron-flex-alignment">
+2.  Include the module(s) that you need:
 
-or:
+        <!-- include classes in the main document -->
+        <style is="custom-style" include="iron-flex iron-flex-alignment">
 
-    <!-- import classes in an element -->
-    <style include="iron-flex iron-flex-alignment">
+    Or:
 
-Then simply apply the classes to any element.
+        <!-- import classes in an element -->
+        <style include="iron-flex iron-flex-alignment">
```

Comment:
This tag should be closed with `</style>`. I found this out the hard way that it broke the css parsing of Polymer inside templates.